lib/aamva/request/authentication_token_request.rb

```diff
@@ -34,15 +34,15 @@ def initialize(
 
       def send
         Response::AuthenticationTokenResponse.new(
-          http_client.post(url, body, headers)
+          http_client.post(url, body, headers),
         )
-      rescue Faraday::TimeoutError, Faraday::ConnectionFailed => err
-        message = "AAMVA raised #{err.class} waiting for authentication token response: #{err.message}"
+      rescue Faraday::TimeoutError, Faraday::ConnectionFailed => e
+        message = "AAMVA raised #{e.class} waiting for authentication token response: #{e.message}"
         raise ::Proofer::TimeoutError, message
       end
 
       def self.auth_url
-        Env.fetch('AUTH_URL', DEFAULT_AUTH_URL)
+        Env.fetch('AAMVA_AUTH_URL', DEFAULT_AUTH_URL)
```

Comment:
should we have it also fall back on the old name and warn?

Reply:
We can but we actually don't use the old name anywhere. And if we did change it it wouldn't work. The default is hardcoded somewhere else in the code. PR to fix that is coming up next.

Reply:
Ok sounds good!